scout/server/links.py

```diff
@@ -3,6 +3,8 @@
 from scout.constants import SPIDEX_HUMAN
 from scout.utils.convert import amino_acid_residue_change_3_to_1
 
+SHALLOW_REFERENCE_STR_LOCI = ["ARX", "HOXA13"]
```

Comment:
Could this list be moved to constants? Makes sense, also in case we need to add some more?

Reply:
It could definately! I kept it as its currently only used in one file, so made more sense to have it at hand. Im ok with either!

Reply:
If all else is equal, laziness wins? 😉